cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/PhaseTapChangerConversion.java

```diff
@@ -313,33 +317,37 @@ private void fillAlphaRhoListsSymmetrical(
         if (stepPhaseShiftIncrementIsSet && stepPhaseShiftIncrement != 0) {
             for (int step = lowStep; step <= highStep; step++) {
                 int n = step - neutralStep;
-                double alpha = n * Math.toRadians(
-                    (configIsInvertVoltageStepIncrementOutOfPhase ? -1 : 1)
-                        * stepPhaseShiftIncrement);
-                double rho = 1.0;
+                double angle = n * Math.toRadians(
+                        (configIsInvertVoltageStepIncrementOutOfPhase ? -1 : 1)
+                                * stepPhaseShiftIncrement);
 
                 // CGMES uses ratio to define the relationship between voltage ends while IIDM uses rho
                 // ratio and rho as complex numbers are reciprocals. Given V1 and V2 the complex voltages at end 1 and end 2 of a branch we have:
                 // V2 = V1 * rho and V2 = V1 / ratio
                 // This is why we have: rho=1/ratio and alpha=-angle
-                alphas.add(-alpha);
-                rhos.add(1 / rho);
+                double alpha = -angle;
+
+                // In IIDM, all PTC must be side one
+                alphas.add(side == 1 ? alpha : -alpha);
+                rhos.add(1.0);
             }
         } else {
             for (int step = lowStep; step <= highStep; step++) {
                 int n = step - neutralStep;
                 double dy = (n * du / 2 - du0) * Math.sin(theta);
-                double alpha = 2 * Math.asin(dy);
-                double rho = 1.0;
+                double angle = 2 * Math.asin(dy);
```

Comment:
I think it is better to leave ratio = 1.0; for a better understanding of the code.